projects/mcp-server.md

```diff
@@ -0,0 +1,136 @@
+
+* [pavolloffay/opentelemetry-mcp-server](https://github.com/pavolloffay/opentelemetry-mcp-server): Focuses on collector configuration.
+* [austinlparker/otel-mcp](https://github.com/austinlparker/otel-mcp): Handles collector configuration and data profiling.
+* [mottibec/otelcol-mcp](https://github.com/mottibec/otelcol-mcp): Focuses on collector configuration.
+* [shiftyp/otel-mcp-server](https://github.com/shiftyp/otel-mcp-server): Provides data profiling, but requires OpenSearch.
+* [liatrio-labs/otel-instrumentation-mcp](https://github.com/liatrio-labs/otel-instrumentation-mcp): Manages instrumentation.
+* [traceloop/opentelemetry-mcp-server](https://github.com/traceloop/opentelemetry-mcp-server): Provides data profiling by connecting to Jaeger, Tempo and Traceloop.
+
+Each of these servers uses a different approach, particularly for collector configuration and data profiling.
+This fragmentation creates confusion for users in terms of installation and configuration. It is less effective
+as multiple competing tools fill up context window and provide overlapping functionality.
+
+### Project Scope and Architecture
+
+The scope of this project is to create OpenTelemetry MCP server(s) to simplify deployment and day-2 operations.
+The MCP server should also provide data profiling/intelligence capabilities to support the day-2 operation use-cases.
+
+### Goals, objectives, and requirements
+
+The goals are divided into categories. However, some common goals apply to all created MCP(s):
+
+* A common installation and configuration
+
+#### Collector
+
+* Deployment, configuration and management
+* Simplify writing OpenTelemetry Transformation Language (OTTL)
+* Simplify writing PII rules based on the received data
+
+#### Instrumentation
+
+* SDK configuration
+* Auto-instrumentation configuration
+* Identify instrumentation issues: single span traces, broken traces, high cardinality attributes
+
+#### Semantic conventions
+
+* Weaver schema generation
+* Context optimized querying of the official semantic conventions registry
+
+## Deliverables
+
+* Collector MCP server
+  * Configuration use-cases
+  * Data profiling use-cases: writing PII rules, high cardinality attributes, broken traces, single span traces
+* Standalone MCP Server
+  * Instrumentation use-cases
+  * Collector provisioning and configuration use-cases
+  * Understanding changes in released artifacts
+
+## Staffing / Help Wanted
+
+This project requires a blend of OpenTelemetry collector, documentation and instrumentation expertise and expertise in building MCP server(s).
+
+### SIG
+
+This project will be led by a cross-cutting effort coordinating with the Collector, documentation, and language SIGs. 
+
+This effort will be hosted in the existing Developer Experience SIG.
+
+### Required staffing
+
+#### Project Leads(s)
+
+* [@pavolloffay](https://github.com/pavolloffay) (Red Hat)
+* [@niwoerner](https://github.com/niwoerner) (OllyGarden)
+
+#### TC Sponsor
+
+Not needed.
```

Comment:
DevEx SIG should have a TC point of contact (at least "escalating"), if they don't have one right now, this is not part of this proposal and blocking but something that should be resolved.